web/forms/implementing_simple_autocompletion/my_element.html

```diff
@@ -0,0 +1,26 @@
+<link rel="import" href="match_element.html">
+<link rel="import" href="packages/core_elements/core_input.html">
+<polymer-element name="my-element">
+  <template>
+    <input is="core-input" on-keyup="{{findCandidates}}" on-keydown="{{keyDown}}" placeholder="Type slowly">
+    <template if="{{!matchingCandidates.isEmpty}}">
+      <ul class="candidate-list">
+        <template repeat="{{candidate in matchingCandidates}}">
+          <li>
+            <match-element class="{{ { 'selected': candidate == selectedCandidate } }}"
+                           value="[[candidate]]"
+                           inputQuery="[[inputQuery]]"></match-element>
+          </li>
+          <style>
```

Comment:
this style should be moved up to just inside the first template tag, where it is now it will get copied for every match candidate.